app/views/pages/crayons.html.erb

```diff
@@ -0,0 +1,886 @@
+<% title "Crayons" %>
```

Comment:
This page is meant to help preview and test all the components etc... Hopefully, everything from here will eventually land in storybook.

Reply:
For the crayons page in the app, why don’t we just put it in Storybook right away?